File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int cnt = 0;
        for (String detail : details) {
            int age = Integer.parseInt(detail.substring(detail.length()-4, detail.length()-2));
            if (age > 60)
                cnt++;
        }
        return cnt;
    }
}

File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.js
/**
 * @param {string[]} details
 * @return {number}
 */
var countSeniors = function(details) {
    let cnt = 0;
    for (let detail of details) {
        if (Number(detail.slice(-4, -2)) > 60) {
            cnt++;
        }
    }
    return cnt;
};